app/main.py

```diff
@@ -1 +1,52 @@
-#  write your code here
+class Animal:
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True) -> None:
```

Comment:
if you're using "(" on separate row, you should use "): -> None" on separate row too.